x/mint/spec/04_params.md

```diff
@@ -4,41 +4,33 @@ order: 4
 
 # Parameters
 
-The mint module contains the following parameters:
+The `mint` module contains the following parameters:
 
 | Key                  | Type                | Example |
 |----------------------|---------------------|---------|
 | mint_denom           | string              | "stake" |
 | block_time_threshold | time.duration       | "10s"   |
 | inflation_schedules  | []InflationSchedule |         |
 
-
 ## MintDenom
 
 MintDenom is the denomination of the coin to be minted.
 
 ## BlockTimeThreshold
 
-If the difference in block time from the `LastBlockTime` is greater than `BlockTimeThreshold`, the block only generates inflation as long as `BlockTimeThreshold.
-so actual minted amount could be less than the defined `InflationSchedule.Amount` depending on the number of times blocks having the block time length over `BlockTimeThreshold` occurs.
-
-This is to prevent inflationary manipulation attacks caused by stopping chains or block time manipulation.
+It is a parameter to prevent from inflationary manipulation attacks. Although it is highly unlikely to happen, block time manipulation can be possible with a group of malicious validators who have enough voting power. In case block time delays, BlockTimeThreshold will be used for current block time to calculate inflation amount for the block. If the `LastBlockTime` is greater than `BlockTimeThreshold`, the block generates with `BlockTimeThreshold` value. Therefore, it is possible that the actual minted amount is less than the defined inflation schedule amount.
 
 ## InflationSchedules
 
-InflationSchedules is a list of `InflationSchedules`, Those inflation schedules cannot overlap, start time is inclusive for the schedule and end time is exclusive so the end times and other start times could be the same.
-
-`InflationSchedule` defines the start and end time of the inflation period, and the amount of inflation during that period.
-
-`InflationSchedule.Amount` should be over the inflation schedule duration seconds to avoid decimal loss
```

Comment:
Is there a reason why these contents were omitted? This is important information for parameter setting.

Reply:
Let me revert that information.

> `InflationSchedule.Amount` should be over the inflation schedule duration seconds to avoid decimal loss

In this sentence, is `the inflation schedule duration seconds` referring to `EndTIme - StartTime`?

Reply:
> Those inflation schedules cannot overlap, start time is inclusive for the schedule and end time is exclusive so the end times and other start times could be the same.

Can you elaborate on this part? I think using `inclusive` and `exclusive` provide ambiguity in the sentence. Does that mean start time for each inflation schedule is inclusive of the current time? (it can be greater than or equal to the current time)?

Reply:
Thanks for the explanation. I paraphrased it to cover all the stuff that was previously stated. If there is anything you would like to add or update, let me know.

> It is a list of inflation schedules to mint coins to be sent to the fee collector account. Each `InflationSchedule` defines start time, end time, and an amount of inflation. The start and end times of inflation schedules can't overlap from one another. While `StartTime` is inclusive of the current time, `EndTime` is exclusive. And, `Amount` for each schedule must be greater than the amount that is converted by subtracting the `EndTime` from the `StartTime` in seconds. Reference the [validate function](https://github.com/cosmosquad-labs/squad/blob/83e40f9204e1554cf3a0f45089fba71bb5685c70/x/mint/types/params.go#L104-L126) to fully understand them.

Reply:
Looks good, how about removing the link of the last sentence because the link may change later

> Reference the validate function on `squad/x/mint/types/params.go` to fully understand them.